bpf/process/types/basic.h

```diff
@@ -740,6 +740,49 @@ filter_char_buf_prefix(struct selector_arg_filter *filter, char *arg_str, uint a
 	return !!pass;
 }
 
+static inline __attribute__((always_inline)) void
+copy_reverse(__u8 *dest, uint len, __u8 *src)
+{
+	uint i;
+
+	len &= (STRING_POSTFIX_MAX_LENGTH - 1);
```

Comment:
nit: might be worth defining `STRING_POSTFIX_MAX_MASK = STRING_POSTFIX_MAX_LENGTH` since we seem to be using it a lot.